.github/workflows/auto-publish.yml

```diff
@@ -1,14 +1,14 @@
 # This workflow will upload a Python Package using Twine when a release is created
 # For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
 
-name: Upload Package to PyPI
+name: Upload Package to PyPI and Publish NWBPanel Docker Image
 
 on:
   release:
     types: [published]
 
 jobs:
-  deploy:
+  publish-pypi:
```

Comment:
Highly recommend setting this to `push` for a commit or 2 on this PR to test/ensure that it works before merging it to main, otherwise it's going to be a hassle to debug

Reply:
(and then remove `push` before merging, of course)

Reply:
I agree, will do that